CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 2.2.0-beta4
+- Support of performance collection and live metrics for Azure Web Apps is enabled.
+- Custom performance metric support is not supported for Azure Web Apps.
```

Comment:
I just want to make sure that they cannot specify any performance counter, because we only have a subset of the tracked counters. I will remove this from here, and will add these details to git documentation. Thanks